services/scabbard/libscabbard/src/service/shared.rs

```diff
@@ -87,14 +91,18 @@ impl ScabbardShared {
         .expect("String -> PeerId -> String conversion should not fail");
 
         // initialize pending_batches metric
-        gauge!("splinter.scabbard.pending_batches", 0);
+        gauge!("splinter.scabbard.pending_batches", 0,
+                "circuit_id" => circuit_id.clone()
```

Comment:
This should actually be `"service" => format!("{}::{}", circuit_id, service_id)`